pkg/operator/operator.go

```diff
@@ -305,13 +308,21 @@ func newReportingOperator(
 		reporting.NewReportListerGetter(reportInformer.Lister()),
 	)
 
+	logger.Infof("setting up event broadcasters")
+	utilruntime.Must(meteringv1scheme.AddToScheme(scheme.Scheme))
```

Comment:
a)`Must` is a candidate for worst function name ever; b) why are we wrapping with it? Like why do we want to cause a panic there, how will it help matters?

Reply:
Could rename `Must` to `PanicOnError` and delete its comment.

Reply:
Yeah agreed on the poor function name selection. But to answer your question, `AddToScheme(...)` returns an error, but the Reporting constructor function does not, so I thought a panic would be applicable here instead of ignoring that error.

Reply:
Not our func, nvm